pages/index.js

```diff
@@ -1,3 +1,10 @@
+import EventList from "@/components/EventList/EventList";
+import { events } from "@/lib/events";
+
 export default function HomePage() {
-  return <div></div>;
+  return (
+    <>
+      <EventList events={events} />
+    </>
```

Comment:
```suggestion
      <EventList events={events} />
```

If there is only one element being returned you don't need the fragments :)